source/frontend/src/features/mapSideBar/management/tabs/ManagementFileTabs.tsx

```diff
@@ -52,7 +54,13 @@ export const ManagementFileTabs: React.FC<IManagementFileTabsProps> = ({
     name: 'File Details',
   });
 
-  if (managementFile?.id && hasClaim(Claims.DOCUMENT_VIEW)) {
+  tabViews.push({
```

Comment:
Does this view depend on a management file id the way documents and notes does?

Reply:
good question. I was following the same pattern of the management summary tab which doesn't check for a valid management file ID but I can make the change here to be more defensive

Reply:
Made the change to check for management file id